app/javascript/components/charts/AmChartGauge.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div class="am-chart--gauge">
+    <p 
+      v-if="title"
+      class="chart__title"
+    >
+      {{ title }}
+    </p>
+
+    <div class="chart__chart">
+      <div 
+        class="chart__svg"
+        :id="id"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import * as am4core from "@amcharts/amcharts4/core"
+import * as am4charts from "@amcharts/amcharts4/charts"
+import am4themes_animated from "@amcharts/amcharts4/themes/animated"
+import ScrollMagic from 'scrollmagic'
+
+export default {
+  name: 'AmChartGauge',
+
+  props: {
+    id: {
+      required: true,
+      type: String
+    },
+    legendData: {
+      type: Array // [{ name: String, fill: Strin }]
+    },
+    target: {
+      type: Number
+    },
+    theme: {
+      default: '#000',
+      type: String,
+    },
+    title: {
+      String: Number
+    },
+    trigger: { 
+      type: String
+    },
+    value: {
+      required: true,
+      type: Number
+    }
+  },
+
+  date () {
+    return {
+      axis: {},
+      chart: {},
+    }
+  },
+
+  mounted () {
+    this.createChart()
+
+    am4core.useTheme(am4themes_animated)
```

Comment:
Just wondering, is it better to call this in `mounted` or outside of the options API entirely (i.e. along with the imports)?